.drone.star

```diff
@@ -1237,18 +1258,22 @@ def ocisService(type, tika_enabled = False):
         "LDAP_GROUP_SUBSTRING_FILTER_TYPE": "any",
         "LDAP_USER_SUBSTRING_FILTER_TYPE": "any",
         "PROXY_ENABLE_BASIC_AUTH": True,
-        "STORAGE_HOME_DRIVER": "ocis",
-        "STORAGE_METADATA_DRIVER_OCIS_ROOT": "/srv/app/tmp/ocis/storage/metadata",
-        "STORAGE_SHARING_USER_JSON_FILE": "/srv/app/tmp/ocis/shares.json",
-        "STORAGE_USERS_DRIVER": "ocis",
-        "STORAGE_USERS_DRIVER_LOCAL_ROOT": "/srv/app/tmp/ocis/local/root",
-        "STORAGE_USERS_DRIVER_OCIS_ROOT": "/srv/app/tmp/ocis/storage/users",
-        "STORAGE_USERS_DRIVER_OWNCLOUD_DATADIR": "%s" % dir["ocisRevaDataRoot"],
         "WEB_ASSET_PATH": "%s/dist" % dir["web"],
-        "WEB_UI_CONFIG": "%s" % dir["ocisConfig"],
         "FRONTEND_SEARCH_MIN_LENGTH": "2",
         "FRONTEND_OCS_ENABLE_DENIALS": True,
     }
+    if type == "app-provider":
+        environment["GATEWAY_GRPC_ADDR"] = "0.0.0.0:9142"
+        environment["MICRO_REGISTRY"] = "mdns"
+    else:
+        environment["WEB_UI_CONFIG"] = "%s" % dir["ocisConfig"]
```

Comment:
will web_config also affect app-provider?

Reply:
no it did not work https://drone.owncloud.com/owncloud/web/38160/12/18